### ISSUE TITLE
UI session stops working after 500 (Internal Server Error)

### ISSUE THREAD

**Issue by HiromuHota** [HiromuHota/pentaho-kettle#92]
By "UI session stops working," I mean any menu, button, everything becomes unresponsive.
For example, clicking on File menu only shows ... as shown in the screenshot below.

<img width="719" alt="screen shot 2018-01-24 at 10 28 56 am" src="https://user-images.githubusercontent.com/20668349/35309849-c22ceb6c-00f1-11e8-8809-5dde3f1c5552.png">

How to reproduce?
1. Open webSpoon
2. Connect a Repository (tested with File Repository or Pentaho Repository)
3. Click File open icon (or File > Open)
4. Reload webSpoon (in the same browser tab)
5. Connect a Repository (tested with File Repository or Pentaho Repository)
6. Click File open icon (or File > Open)

Which version is affected?
At least 0.8.0.13-full and a223998809169d61c1454d3074d4d54e693a89ac

**Comment by HiromuHota** [comment]
The following error messages were captured in the browser side (Developer Tools)

```
HTTP Status 500 - Remote object already registered, id: w277
type Exception report

message Remote object already registered, id: w277

description The server encountered an internal error that prevented it from fulfilling this request.

exception

java.lang.IllegalArgumentException: Remote object already registered, id: w277
	org.eclipse.rap.rwt.internal.remote.RemoteObjectRegistry.register(RemoteObjectRegistry.java:46)
	org.eclipse.rap.rwt.internal.protocol.RemoteObjectFactory.createForId(RemoteObjectFactory.java:60)
	org.eclipse.rap.rwt.internal.protocol.RemoteObjectFactory.createRemoteObject(RemoteObjectFactory.java:49)
	org.eclipse.swt.internal.widgets.shellkit.ShellLCA.renderInitialization(ShellLCA.java:102)
	org.eclipse.swt.internal.widgets.shellkit.ShellLCA.renderInitialization(ShellLCA.java:1)
	org.eclipse.rap.rwt.internal.lifecycle.WidgetLCA.render(WidgetLCA.java:38)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA$RenderVisitor.render(DisplayLCA.java:318)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA$RenderVisitor.visit(DisplayLCA.java:302)
	org.eclipse.swt.internal.widgets.WidgetTreeUtil.accept(WidgetTreeUtil.java:32)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA.renderShells(DisplayLCA.java:175)
	org.eclipse.swt.internal.widgets.displaykit.DisplayLCA.render(DisplayLCA.java:101)
	org.eclipse.rap.rwt.internal.lifecycle.Render.execute(Render.java:32)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.continueLifeCycle(RWTLifeCycle.java:155)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.sleep(RWTLifeCycle.java:229)
	org.eclipse.swt.widgets.Display.sleep(Display.java:1258)
	org.eclipse.rap.rwt.application.AbstractEntryPoint.createUI(AbstractEntryPoint.java:70)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.createUI(RWTLifeCycle.java:177)
	org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle$UIThreadController.run(RWTLifeCycle.java:290)
	java.lang.Thread.run(Thread.java:748)
	org.eclipse.rap.rwt.internal.lifecycle.UIThread.run(UIThread.java:107)
```

**Comment by HiromuHota** [comment]
Found that this issue shared the same root cause as https://github.com/HiromuHota/pentaho-kettle/issues/38.

**Comment by HiromuHota** [comment]
Resolved by 712d0b351d81952ff80e6362dc5a5b1b483df8e0.
